notifique/src/main/res/drawable/thumb.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <padding
+        android:paddingLeft="22dp"
+        android:paddingRight="22dp" />
+
+    <solid android:color="@color/primary" />
+
+    <stroke
+        android:width="5dp"
+        android:color="@color/primary" />
```

Comment:
why do we have a solid color and a stroke here?

Reply:
removed - dont need